main/release/utils/aws_arn_helper.py

```diff
@@ -0,0 +1,17 @@
+from boto3 import Session
+
+
+class ArnTopicNotFound(Exception):
+    pass
+
+
+class AwsArnHelper:
+
+    @staticmethod
+    def get_ssm_arn_topic_name(aws_session: Session, parameter_name: str) -> str:
+        parameters = aws_session.client('ssm').get_parameters(Names=[parameter_name])
```

Comment:
For a single parameter using the `get_parameter` method is more straightforward

Reply:
Thank you, I was not aware of this method, the autocomplete is broken for me with these client methods.

Reply:
changes will be provided in the next commit